examples/tick-multi-cluster/main.tf

```diff
@@ -4,9 +4,18 @@
 # balancer in front of the data nodes to handle providing the public interface into the cluster.
 # ---------------------------------------------------------------------------------------------------------------------
 
+# ----------------------------------------------------------------------------------------------------------------------
+# REQUIRE A SPECIFIC TERRAFORM VERSION OR HIGHER
+# This module has been updated with 0.12 syntax, which means it is no longer compatible with any versions below 0.12.
+# ----------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.12"
+}
+
```

Comment:
Can you add the following block here so that it differentiates the `aws` provider setup from `terraform` setup?

```
# ------------------------------------------------------------------------------
# CONFIGURE OUR AWS CONNECTION
# ------------------------------------------------------------------------------
```